lib/hexpm_web/views/package_view.ex

```diff
@@ -142,19 +145,23 @@ defmodule HexpmWeb.PackageView do
   def retirement_html(retirement) do
     reason = ReleaseRetirement.reason_text(retirement.reason)
 
-    cond do
-      reason && retirement.message ->
-        [content_tag(:strong, "Retired package:"), " ", reason, " - ", retirement.message]
+    case retirement.reason do
+      "report" -> [content_tag(:strong, "Marked package:")]
+      _ -> [content_tag(:strong, "Retired package:")]
+    end ++
+      cond do
+        reason && retirement.message ->
+          [" ", reason, " - ", retirement.message]
 
-      reason ->
-        [content_tag(:strong, "Retired package:"), " ", reason]
+        reason ->
+          [" ", reason]
 
-      retirement.message ->
-        [content_tag(:strong, "Retired package:"), " ", retirement.message]
+        retirement.message ->
+          [" ", retirement.message]
 
-      true ->
-        [content_tag(:strong, "Retired package")]
-    end
+        true ->
+          []
+      end
```

Comment:
This is a bit hard to read with `case ... ++ cond ...`. I suggest splitting up the expressions and storing the result in variables and then `variable1 ++ variable2`.